docs/pages/Troubleshooting.md

```diff
@@ -188,3 +188,101 @@ The following is a walk-through on how to evaluate the performance impact of an
 1. You can now share these results on GitHub or wherever you might need them.
 
 **Happy benchmarking! 🏎**
+
+
+## ES6+ Limitations
+
+### Summary
+- Not supported ES6+ features: Computed properties, SpreadElement, TemplateLiteral with expression.
```

Comment:
This summary might be quite confusing for readers. It's too oversimplified.
I would recommend to to structure the whole chapter more target group specific.
````
- ES6+ Limitations
  Highlight that we support ES6+ with some small restrictions in special places
    -  ES6+ Limitations for Applications
    -  ES6+ Limitations for Libraries

````

The goal should be that we celebrate ES6+ support 🎊